prac_06/guitar.py

```diff
@@ -0,0 +1,28 @@
+"""Class defining a guitar."""
+
+CURRENT_YEAR = 2021     # Improvement: Use computer's internal clock
+VINTAGE_AGE = 50
+
+
+class Guitar:
+    """Represents a Guitar object."""
+
+    def __init__(self, name="", year=0, cost=0):
+        """Initialise a Guitar instance."""
+        self.name = name
+        self.year = int(year)
+        self.cost = float(cost)
+
+    def __str__(self):
+        """String form of the instance."""
```

Comment:
Function docstring could be re-written as """**Return** string form of the instance."""

Reply:
Missed that. Thanks.